atlp-rwanda/e-comm-team-brogrammers-fn
# #184948157 users can login with google

#### What does this PR do?

This will allow users to login with their google account after click google login button.

#### Description of Task to be completed?
On the Login route, users should also be able to Login via Google or Facebook Button.
#### How should this be manually tested?
- To test this clone the repo run `npm install && npm run start`
- In the browser visite `/login/google` and click the `google login button` you shall see.
- After logging in you should be redirected to home.
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker/Trello stories?
https://www.pivotaltracker.com/n/projects/2639400/stories/184948157
#### Screenshots (if appropriate)
<img width="902" alt="Screenshot 2023-04-20 at 16 58 26" src="https://user-images.githubusercontent.com/57767759/233406128-3ce8878b-dd9a-4368-998f-edffe6cca1d7.png">
<img width="1004" alt="Screenshot 2023-04-24 at 15 58 09" src="https://user-images.githubusercontent.com/57767759/234018954-6add06f2-a615-4848-baf4-70be10444810.png">


#### Questions:
n/a
